app/src/main/kotlin/com/simplemobiletools/calculator/activities/DrawActivity.java

```diff
@@ -43,4 +59,31 @@ public void onClick(View v) {
     public void supersedeSketchSheetView(SketchSheetView equationSketchView) {
         this.equationSketchView = equationSketchView;
     }
+
+    public void toggleWebView(@Nullable String uri){
```

Comment:
Some inconsistent style in this code section. Missing a space before "{" and after "if" and "else".

Reply:
Fixed